### DISCUSSION TITLE
server.growatt.com Registration Code

### DISCUSSION THREAD

**Discussion by macfloripa** [johanmeijer/grott#75] [Q&A]
I tried to login into the growatt server (desktop) and they are asking for a Registration Code. Any ideas on how to get this? I've sent an email to growatt, but no response thus far.

**Comment by johanmeijer** [comment]
Can you send a screen print. It is a long time ago that I registered with Growatt. Maybe if I see it I can remember.

**Comment by macfloripa** [comment]
It's back 'working' (with the usual errors) today. I think they did an upgrade to the site and that was the cause of the problem.
